app/routes/formbuilder/edit.js

```diff
@@ -70,6 +71,11 @@ export default class FormbuilderEditRoute extends Route {
           'http://lblod.data.gift/display-types/countryCodeConceptSchemeSelector',
         edit: CountryCodeConceptSchemeSelectorComponent,
       },
+      {
+        displayType:
+          'http://lblod.data.gift/display-types/storeConceptSchemeSelector',
+        edit: StoreConceptSchemeSelectorComponent,
```

Comment:
The use of a custom component might be a bit troublesome. When the user chooses a dropdown, it will reflect in the outputed code. 

![image](https://github.com/lblod/frontend-form-builder/assets/52280338/d9f2430f-18bb-43f4-be9e-bac44b83b4ee)

I tried to overwrite the component but unfortunatly ember-submission-form-fields does not allow it so we might be a bit stuck here.

Reply:
If we..

The code in the editor will always show the custom component `StoreConceptSchemeSelector`. If we add somewhere near the editor a button to export the code we than can overwrite every custom component if we want but not sure if this is something we want to do